### PR TITLE
Mdvm updates

### PR DESCRIPTION
**Pull request by anders-alex**
Required items, please complete
   
   Change(s):
- Function App Code
    - Added more efficient PowerShell module to handle sending data to Azure Monitor. Removed unneeded async complexity.
    - Updated .Net libraries to recent versions.
    - Resolved intermittent Azure Monitor HTTP 400 error during high/concurrent loads.
    - Added version.info file to track build version.
    - Removed error action preference of "stop" (code will continue to execute on non-terminating errors).
- Deployment
    - Updated to included private network option by default and includes non-private network option in the same template.
    - Updated MDVMCVEKB_CL table schema to include the cveSupportability property.
    - Added custom role so Managed Identity only has read access to MDVM tables.
    - Added option to deploy workbooks as part of initial template.
    - Added option to enabled Function App Elastic Premium plan.
    - Removed need for supplying Log Analytics location in addition to resource ID.
    - Added DeploymentVersion configuration property to App Service to track current version.
    - Overall simplification and cleanup.
    - All tables now use the default retention period of the workspace.

   Reason for Change(s):
   - Fixed bugs and added new functionality.

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

File: DataConnectors/M365Defender-VulnerabilityManagement/azureDeploy.json [hunk 1]
```diff
@@ -319,211 +290,114 @@
               "name": "WEBSITE_RUN_FROM_PACKAGE",
               "value": "1"
```
Comment (v-prasadboke) [inline DataConnectors/M365Defender-VulnerabilityManagement/azureDeploy.json:291]: The value is set to 1, will it fetch zip from repo ?

**Comment by v-prasadboke** [comment]
Also can you share output/working images of Function app

File: DataConnectors/M365Defender-VulnerabilityManagement/azureDeploy.json (hunk 1 above)
Comment (anders-alex) [reply DataConnectors/M365Defender-VulnerabilityManagement/azureDeploy.json:291]: Correct.

**Comment by anders-alex** [comment]
@v-prasadboke - please see below:

image

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**